src/css/app.less

```diff
@@ -102,10 +102,13 @@ body {
         }
     }
 
-    .email-form {
-        .success-message {
-            .h3;
-            color: @accent-text-color;
+    @media @md-up {
```

Comment:
Was ist `md-up`?

Reply:
Auf Deutsch? 😆Lol, not going to try.

It’s the breakpoint for medium sized devices and up (> 992 pixels), which maps to Bootstrap’s “md” column size.

Reply:
Goodness 😅